### ISSUE TITLE
Bug: I can't create a username

### ISSUE THREAD

**Issue by mark-hahn** [kochrt/cascade.page#20]
Clicking on the Create Username button doesn't do anything.  This makes cascade pretty much useless for me.

Chrome: Version 96.0.4664.110 (Official Build) (64-bit).

I'll take a look at the code

**Comment by mark-hahn** [comment]
Update:  I couldn't debug from the cascade.page web page.  Something is keeping the vue dev extension from running (production build?).

I cloned the repo and I am running cascade on my localhost.  However, firebase is blocking auth from my machine so I can't debug the username.  I would be happy to continue debugging if anyone has any suggestion on how I can continue.

I'll continue staring at the code but I doubt I will be able to find anything that way.

**Comment by kochrt** [comment]
I'll check it out. In the meantime can you try hitting enter instead of the button after inputting your username and see if that works?

**Comment by mark-hahn** [comment]
How do I get the input field?  Clicking doesn't do anything.

**Comment by mark-hahn** [comment]
I figured out how to get past the problem.  I opened up the inspector by right-clicking on the button.  It had a `disabled` property set.  After removing that property I was able to click on the button and the pop-up appeared.  After putting in my new username a page appreared with a 404 error but adding my timeline name to the url fixed it.  All in all the code needs some work.  I'd be happy to help if you put up the page in debug mode.
